examples/complete/variables.tf

```diff
@@ -16,14 +20,18 @@ variable "prefix" {
   default     = "complete"
 }
 
-variable "resource_group" {
-  type        = string
-  description = "An existing resource group name to use for this example, if unset a new resource group will be created"
-  default     = null
-}
-
-variable "resource_tags" {
+variable "tags" {
   type        = list(string)
   description = "Optional list of tags to be added to created resources"
   default     = []
 }
+
+variable "region" {
+  type        = string
+  description = "IBM Cloud region where event notification will be created, supported regions are: us-south (Dallas), eu-gb (London), eu-de (Frankfurt), au-syd (Sydney)"
+  default     = "us-south"
+  validation {
+    condition     = contains(["us-south", "eu-gb", "eu-de", "au-syd"], var.region)
+    error_message = "The specified region is not valid, supported regions are: us-south (Dallas), eu-gb (London), eu-de (Frankfurt), au-syd (Sydney)"
+  }
+}
```

Comment:
no need to duplicate the validation here. It will use the validation from the root module. Means we don't have to main it in 2 places

Reply:
Done.